src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -47,7 +50,55 @@ public PersonCard(Person person, int displayedIndex) {
         phone.setText(person.getPhone().value);
         address.setText(person.getAddress().value);
         email.setText(person.getEmail().value);
-        person.getTags().forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        labelTag(person);
+    }
+
+    /**
+     * Set the label for the tag
+     * @param p
+     */
+    private void labelTag(Person p) {
+
```

Comment:
Extra blank line. Use blank lines sparingly. It's good to have blank lines to separate logical code blocks, but too many blank lines will increase the height of the code, which increases the amount of scrolling.